src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -855,7 +855,9 @@ FixedwingPositionControl::control_position(const hrt_abstime &now, const Vector2
 						   tecs_fw_thr_max,
 						   tecs_fw_mission_throttle,
 						   false,
-						   radians(_param_fw_p_lim_min.get()));
+						   radians(_param_fw_p_lim_min.get()),
+						   _param_climbrate_target.get(),
```

Comment:
Are you always passing in the same climbrate and sink rate?

Reply:
@dagar Sorry, I already got rid of this but forgot to push. One sec.